### PR TITLE
fix: root typesync

### PR DESCRIPTION
**Pull request by lishaduck**
Applies suggestion from #295, so I can check CI.

Commit 1 should fail, then I'll push commit 2, which should pass if all goes well.

### CODE REVIEW

**Comment by lishaduck** [comment]
Superseded by #312.

File: apps/sheriff-webservices/package.json [hunk 1]
```diff
@@ -8,16 +8,15 @@
...
     "clean": "rm -rf .turbo dist node_modules/.cache"
   },
   "devDependencies": {
     "@sherifforg/types": "workspace:*",
     "@tsconfig/node-lts": "^18.12.5",
-    "@types/eslint-plugin-jsx-a11y": "^6.9.0",
-    "@types/eslint__js": "^8.42.3",
-    "@types/lodash": "^4.17.6",
+    "@types/eslint-plugin-jsx-a11y": "~6.9.0",
+    "@types/eslint__js": "~8.42.3",
```
Comment (AndreaPontrandolfo) [inline apps/sheriff-webservices/package.json:18]: oh damn, this is old stuff, we don't need this anymore

File: packages/eslint-config-sheriff/package.json [hunk 2]
```diff
@@ -97,9 +97,10 @@
   "devDependencies": {
     "@sherifforg/constants": "workspace:*",
     "@sherifforg/types": "workspace:*",
-    "@types/confusing-browser-globals": "^1.0.3",
-    "@types/eslint-plugin-jsx-a11y": "^6.9.0",
-    "@types/eslint__js": "^8.42.3",
+    "@types/confusing-browser-globals": "~1.0.3",
+    "@types/eslint-plugin-jsx-a11y": "~6.9.0",
+    "@types/eslint__js": "~8.42.3",
+    "@types/lodash": "~4.17.13",
```
Comment (AndreaPontrandolfo) [inline packages/eslint-config-sheriff/package.json:103]: there is lodash in here. Why are we removing the corresponding types?

File: apps/sheriff-webservices/package.json (hunk 1 above)
Comment (lishaduck) [reply apps/sheriff-webservices/package.json:18]: They're dropped in master.

**Comment by AndreaPontrandolfo** [comment]
@lishaduck sorry commented on the wrong pr

**Comment by lishaduck** [comment]
> @lishaduck sorry commented on the wrong pr

Yeah, I was wondering :)

**Pull request closed without merging**